modules/centrifuge/centrifuge/main.nf

```diff
@@ -0,0 +1,39 @@
+process CENTRIFUGE_CENTRIFUGE {
```

Comment:
are there more than one centrifuge tools that you have `CENTRIFUGE_CENTRIFUGE` as the process name rather than `CENTRIFUGE` ?

Reply:
It is only one tool, CENTRIFUGE. I followed the same format as [kraken2](https://github.com/nf-core/modules/tree/master/modules/kraken2/kraken2) module.

Reply:
I used the following command: `nf-core modules create centrifuge/centrifuge` and the process was named that way by default, I did not change it manually.

Reply:
ah yours is a single tools so you just need `nf-core modules create centrifuge`

Reply:
`nf-core modules create <tool>/<sub-tool>` is when a tool has multiple subtools

Reply:
indeed! that needs fixing....will work on it....thanks for bringing this up

Reply:
The PR is now updated.